custom_components/test/fuzzing/fuzzer_utils/ValuePoolFuzzer.py

```diff
@@ -7,7 +7,7 @@ class ValuePoolFuzzer(Fuzzer):
 
     def __int__(self):
         """constructor"""
-        pass
+        self.value_pool = ValuePool()
```

Comment:
@jonathanheitzmann  `value_pool` must be created as a class variable. `valid_types{}`can't access `value_pool`

Reply:
fixed it in [ab57562](https://github.com/ThorbenCarl/PyLoxone/pull/2/commits/ab57562d1b18276c26b8a5a47e89b4345948f071)